spec/whois.md

```diff
@@ -1,90 +1,33 @@
-verifiable credentials to its graduate students. A graduate from the school
-submits as part of their job application to a company a [[ref: Verifiable
-Presentation]] derived from the verifiable credential they received from the
-school. The company receiving the presentation can verify the cryptography of
-the presentation, but can they trust the school that issued the verifiable
-credential? If the school issued the verifiable credential using its `did:webs`
-DID, the company can resolve the DID. It can also resolve the DID's
-`whois` DID URL where it might find VCs from issuers it trusts with the
-`did:webs` DID as the subject. For example:
-
-- A verifiable credential issued by the Legal Entity Registrar for the
-  jurisdiction in which the school is headquartered.
-  - Since the company knows about the Legal Entity Registrar, they can automate
-    this lookup to get more information about the school from the verifiable
-    credential -- its legal name, when it was registered, contact information,
-    etc.
-- A verifiable credential issued by the "Association of Colleges and
-  Universities" for the jurisdiction of the school.
-  - Perhaps the company does not know about the Association for that
-    jurisdiction. The company can repeat the `whois` resolution process for the
-    issuer of _that_ credential. The company might (for example), resolve and
-    verify the `did:webs` DID for the Association, and then resolve the `whois`
-    DID URL to find a verifiable credential issued by the government of the
-    jurisdiction. The company recognizes and trusts that government's authority,
-    and so can decide to recognize and trust the Association.
-- A verifiable credential issued by US News magazine about the school's
-  placement on the [US News Rankings of Colleges and Universities].
-
-Such checks can all be done with a handful of HTTPS requests and the processing
-of the DIDs and verifiable presentations. The result is an efficient, verifiable,
-credential-based, decentralized, multi-domain trust registry.
-
-[US News Rankings of Colleges and Universities]: https://www.usnews.com/education/best-global-universities
+This convention enables those that receive the DID to retrieve and verify the verifiable presentation (and embedded Verifiable Credentials) the DID controller has decided to publish about itself. The intention is that anyone interested in a particular `did:webs` DID can see if there is a `whois` Verifiable Presentation, and if so, if it contains any useful (to the resolver) Verifiable Credentials that might help in learning more about who is the controller of the DID.
 
 ### `whois` DID URL Conventions
+1. The `whois` object, if it exists, MUST be directly beside the `did.json` file, the DID Document for the DID.
+1. The [[ref: DID URL path]] to the folder MUST be `<did:webs DID>/whois`.
+1. The HTTPS path to the `whois` folder MUST be `<HTTPS URL conversion of did:webs identifier>/whois.vp`.
```

Comment:
shouldn't this be the "`whois` verifiable presentation" rather than the folder (which appears in the previous number too?